src/theme/theme.ts

```diff
@@ -99,6 +99,29 @@ const themeOptions: ThemeOptions = {
         },
       },
     },
+    MuiButtonBase: {
+      defaultProps: {
+        disableRipple: true,
+      },
+    },
+    MuiButton: {
+      styleOverrides: {
+        root: {
+          '&:hover': {
+            boxShadow: 'none',
+          },
+        },
+      },
+    },
+    MuiMenuItem: {
+      styleOverrides: {
+        root: {
+          '&.Mui-selected': {
+            backgroundColor: '#954661', //CollageRed[500]
```

Comment:
```suggestion
            backgroundColor: palette.collageRed[500],
```
You can use palette here directly since it's created above this object

Reply:
Initially, I tried that approach, but I received a warning stating, "palette might be undefined." I believe this is because the palette hasn't fully initialized at that moment.

Reply:
@darkares23 oh, adjust the type then. 

```javascript
  interface CustomPaletteOptions {
    collageRed: CustomColorShades;
    collageYellow: CustomColorShades;
    collageOrange: CustomColorShades;
    collageViolet: CustomColorShades;
    collageTeal: CustomColorShades;
    uiGray: CustomColorShades;
    uiCoolGray: CustomColorShades;
    uiWhite: CustomColorShades;
    uiBlue: CustomColorShades;
    uiGreen: CustomColorShades;
    uiYellow: CustomColorShades;
    uiRed: CustomColorShades;
  }
```

Reply:
If I do that I have a different error asking me to add all the properties to the palette:
```
[{
	"resource": "/Users/juan/collage_group/mosaic/src/theme/theme.ts",
	"owner": "typescript",
	"code": "2740",
	"severity": 8,
	"message": "Type '{}' is missing the following properties from type 'CustomPaletteOptions': collageRed, collageYellow, collageOrange, collageViolet, and 8 more.",
	"source": "ts",
	"startLineNumber": 22,
	"startColumn": 9,
	"endLineNumber": 22,
	"endColumn": 16
}]
```

Reply:
You can achieve the result removing all the object in line 116 and on line 40, inside the pallete object add the modification of the primary color as follows 

export const palette: PaletteOptions = {
  ...generatePalette(ui, 'ui'),
  ...generatePalette(collage, 'collage'),
  **primary: { main: collage.red[500] },**
};

Reply:
@diego20 @darkares23 Diego's point makes sense here, we don't have to touch the palette if you change MUI's primary color. Primary will change the color without you setting 'backgroundColor` manually.

Reply:
Ok Juan, I just spoke to Yassee about this, what should be the primary color (since we still have material's blue by default), what should be the selected color and everything. She handed me out a palette of colors to replace the default. No custom selected color for now, just the main colors and let Material handle the variations of them.
This is the palette:
`palette: {
    mode: 'light',
    primary: {
      main: '#954661',
      dark: '#7b1839',
      light: '#a7647a',
    },
    secondary: {
      main: '#c16f8b',
      dark: '#b14b6e',
      light: '#cb869e',
    },
    error: {
      main: '#CC3333',
      light: '#d65c5c',
      dark: '#ba2e2e',
    },
    warning: {
      main: '#d5902f',
      light: '#ea9e34',
      dark: '#a67025',
    },
    success: {
      main: '#18914c',
      light: '#46a770',
      dark: '#168445',
    },
    text: {
      primary: '#333333',
      secondary: '#5c5c5c',
      disabled: '#b4b4b4',
      hint: '#454C55',
    },
  },`
  You can just declare the object and add it where I mentioned and that would be all